blog.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Blog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="./css/styles.css" />
+  </head>
+  <body>
+    <div class="page-wrapper">
+      <header class="heder-container">
+        <a class="logo" href="./index.html">
+          <img src="./images/logo.svg" alt="LOGO"
+        /></a>
+        <nav class="header-nav">
```

Comment:
Logically `header-nav-container` should wrap the `header-nav` because the meaning for 'container' is that it's something that wraps its child.
Another question is 'why do sew use containers?'. Usually, it's needed if we want to apply some styling on its child, some paddings or additional margins. So be conscious about the naming.,